src/nl/hannahsten/texifyidea/lang/commands/LatexGenericRegularCommand.kt

```diff
@@ -143,7 +143,7 @@ enum class LatexGenericRegularCommand(
     INCLUDEFROM("includefrom", RequiredFolderArgument("absolute path"), RequiredFileArgument("filename", false, false, "tex"), dependency = LatexPackage.IMPORT),
     INPUT("input", RequiredFileArgument("sourcefile", true, false, "tex")),
     INPUTFROM("inputfrom", RequiredFolderArgument("absolute path"), RequiredFileArgument("filename", false, false, "tex"), dependency = LatexPackage.IMPORT),
-    INCLUDEGRAPHICS("includegraphics", "key-val-list".asOptional(), RequiredPicturePathArgument("imagefile", true, false, "pdf", "png", "jpg", "eps", "tikz"), dependency = GRAPHICX),
+    INCLUDEGRAPHICS("includegraphics", "key-val-list".asOptional(), RequiredPicturePathArgument("imagefile", true, false, "png", "pdf", "jpg", "jpeg", "jb2", "eps", "mps", "PNG", "PDF", "JPG", "JPEG", "JB2", "EPS"), dependency = GRAPHICX),
```

Comment:
Are you sure that `texdef -t pdflatex -p graphicx Gin@extensions` outputs png in front of pdf?

Reply:
No, I am not

Reply:
Then let's change it back to the default ordering (assuming we fix the other comment and actually have an ordered list), which for me is

`macro:->.pdf,.png,.jpg,.mps,.jpeg,.jbig2,.jb2,.PDF,.PNG,.JPG,.JPEG,.JBIG2,.JB2,.eps`